src/stores/BreadcrumbsStore.ts

```diff
@@ -22,6 +22,7 @@ import defaultDispatcher from "../dispatcher/dispatcher";
 import { arrayHasDiff } from "../utils/arrays";
 import { isNullOrUndefined } from "matrix-js-sdk/src/utils";
 import { SettingLevel } from "../settings/SettingLevel";
+import {MatrixClientPeg} from '../MatrixClientPeg';
```

Comment:
match brace styles please:
```suggestion
import { MatrixClientPeg } from '../MatrixClientPeg';
```